open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb

```diff
@@ -5,13 +5,1070 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # mobile_test.head(2)
remove mobile_test.head(2), or put it in a new cell
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>